web/pandas/pdeps/0010-required-pyarrow-dependency.md

```diff
@@ -0,0 +1,160 @@
+integration into many areas of pandas. With our roadmap noting that pandas strives for better Apache Arrow
+interoperability [^1] and many projects [^2], within or beyond the Python ecosystem, adopting or interacting with
+the Arrow format, making PyArrow a required dependency provides an additional signal of confidence in the Arrow
+ecosystem to pandas users.
+
+Additionally, requiring PyArrow would simplify the related development within pandas and potentially improve NumPy
+functionality that would be better suited by PyArrow including:
+
+- Avoiding runtime checking if PyArrow is available to perform PyArrow object inference during constructor or indexing operations
+
+- Removing redundant functionality:
+  - fastparquet engine in `read_parquet`
+  - potentially simplifying the `read_csv` logic (needs more investigation)
+
+- Avoiding NumPy object data types more by default for analogous types that have native PyArrow support such as:
+  - decimal
+  - binary
+  - nested types (list or dict data)
+  - strings
+
+Out of this group, strings offer the most advantages for users. They use significantly less memory and are faster:
+
+**Performance:**
+
+```python
+import string
+import random
+
+import pandas as pd
+
+
+def random_string() -> str:
+    return "".join(random.choices(string.printable, k=random.randint(10, 100)))
+
+
+ser_object = pd.Series([random_string() for _ in range(1_000_000)])
+ser_string = ser_object.astype("string[pyarrow]")\
+```
+
+PyArrow backed strings are significantly faster than NumPy object strings:
+
+*str.len*
+
+```python
+In[1]: %timeit ser_object.str.len()
+118 ms ± 260 µs per loop (mean ± std. dev. of 7 runs, 10 loops each)
+
+In[2]: %timeit ser_string.str.len()
+24.2 ms ± 187 µs per loop (mean ± std. dev. of 7 runs, 10 loops each)
+```
+
+*str.startswith*
+
+```python
+In[3]: %timeit ser_object.str.startswith("a")
+136 ms ± 300 µs per loop (mean ± std. dev. of 7 runs, 10 loops each)
+
+In[4]: %timeit ser_string.str.startswith("a")
+11 ms ± 19.8 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)
+```
+
+Another advantage is I/O. PyArrow engines in pandas can provide a significant speedup. Currently, the data
+are cast to NumPy dtypes, which requires roundtripping when converting back to PyArrow strings explicitly, which
+hinders performance.
+
+**Memory**
+
+PyArrow backed strings use significantly less memory. Dask developers investigated this [here](https://www.coiled.io/blog/pyarrow-strings-in-dask-dataframes).
+
+Short summary: PyArrow strings required 1/3 of the original memory.
+
+
+## Drawbacks
+
+Including PyArrow would naturally increase the installation size of pandas. For example, installing pandas and PyArrow
+using pip from wheels, numpy and pandas requires about `70MB`, and including PyArrow requires around `120MB`. An increase
+of installation size would have negative implication using pandas in space-constrained development or deployment environments
+such as AWS Lambda.
```

Comment:
> and including PyArrow requires around `120MB`

maybe I was just misinterpreting this. Could you please rephrase to "and including PyArrow requires an additional `120MB`", to make it unambiguous that "120M" refers to just pyarrow, rather than "pandas + numpy + pyarrow"?